app/src/main/res/layout/activity_session_list.xml

```diff
@@ -57,8 +57,22 @@
             android:id="@+id/text_session_list_progress_update"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_margin="8dp"
+            android:layout_margin="4dp"
             app:layout_constraintTop_toBottomOf="@id/progress_bar_session_list"
+            app:layout_constraintBottom_toTopOf="@+id/text_session_list_progress_update_line_2"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent" />
+
+        <TextView
+            android:id="@+id/text_session_list_progress_update_line_2"
```

Comment:
I think we should use a more descriptive name for this. Maybe like `session_list_progress_step` and `session_list_progress_details`?

Reply:
good point